problem_builder/tests/integration/base_test.py

```diff
@@ -57,7 +60,18 @@ def popup_check(self, mentoring, item_feedbacks, prefix='', do_submit=True):
             self.assertFalse(item_feedback_popup.is_displayed())
 
 
-class MentoringAssessmentBaseTest(MentoringBaseTest):
+class MentoringBaseTest(SeleniumBaseTest, PopupCheckMixin):
+    module_name = __name__
+    default_css_selector = 'div.mentoring'
```

Comment:
@bradenmacdonald niy: moving this before `PopuCheckMixin will reduce the diff size.